filters/glitch/src/GlitchFilter.js

```diff
@@ -0,0 +1,269 @@
+import {vertex} from '@tools/fragments';
+import fragment from './glitch.frag';
+
+/**
+ * The GlitchFilter applies a glitch effect to an object.<br>
+ * ![original](../tools/screenshots/dist/original.png)![filter](../tools/screenshots/dist/glitch.png)
+ *
+ * @class
+ * @extends PIXI.Filter
+ * @memberof PIXI.filters
+ * @param {object} options - The more optional parameters of the filter.
+ * @param {number} [options.slices=5] - The count of slices.
+ * @param {number} [options.offset=100] - The max-offset of slices.
+ * @param {number} [options.direction=0] - The angle in degree of the offset of slices.
+ * @param {number} [options.fillMode=0] - The fill mode of the space after the offset.
+ *                 0: TRANSPARENT; 1: ORIGINAL; 2: LOOP; 3: CLAMP; 4: MIRROR.
+ * @param {number} [options.red=[0,0]] - Red channel offset
+ * @param {number} [options.green=[0,0]] - Green channel offset.
+ * @param {number} [options.blue=[0,0]] - Blue channel offset.
+ */
+export default class GlitchFilter extends PIXI.Filter {
+    constructor(options = {}) {
+
+        super(vertex, fragment);
+
+        const _options = {
+            slices: 5,
+            offset: 100,
+            direction: 0,
+            fillMode: 0,
+            average: false,
+            seed: 0.5,
+            red: [0, 0],
+            green: [0, 0],
+            blue: [0, 0],
+            minSliceWidth: 8,
+            displacementMapSize: 512,
+            displacementMap: null,
+        };
+
+        Object.assign(_options, options);
```

Comment:
Better way to do this is to reassign `options`. Avoids `_options` which is confusing.
```js
options = Object.assign({
    slices: 5,
    offset: 100,
    direction: 0,
    fillMode: 0,
    average: false,
    seed: 0.5,
    red: [0, 0],
    green: [0, 0],
    blue: [0, 0],
    minSliceWidth: 8,
    displacementMapSize: 512,
    displacementMap: null,
}, options);
```

Reply:
DONE